src/Components/Magazine_Block_Controller.php

```diff
@@ -14,44 +14,68 @@ class Magazine_Controller {
 	public function __construct( $block ) {
 		$this->block = (array) $block;
 	}
-	
-	public function get_title(): string {
-		$title = (string) get_field( Magazine_Block::TITLE );
-		
-		return strlen( $title ) < 1 ? '' : $title;
+
+	public function get_attributes(): string {
+		return wp_kses_data( get_block_wrapper_attributes([
+			'class' => implode(' ', [
+				'ucsc-magazine-block',
+				'alignfull',
+				'has-lightest-gray-background-color',
+				'has-global-padding',
+				'is-layout-constrained',
+			]),
+		]) );
 	}
 
-	public function get_overline(): string {
-		$overline = (string) get_field( Magazine_Block::OVERLINE );
+	public function get_title_line_1(): string {
+		$overline = (string) get_field( Magazine_Block::TITLE_LINE_1 );
 
 		return strlen( $overline ) < 1 ? '' : $overline;
 	}
 	
+	public function get_title_line_2(): string {
+		$title = (string) get_field( Magazine_Block::TITLE_LINE_2 );
+		
+		return strlen( $title ) < 1 ? '' : $title;
+	}
+	
+	public function get_subtitle(): string {
+		$subtitle = (string) get_field( Magazine_Block::SUBTITLE );
+		
+		return strlen( $subtitle ) < 1 ? '' : $subtitle;
+	}
+
 	public function get_magazines(): array {
 		$magazines = (array) get_field( Magazine_Block::ITEMS );
 		
 		if ( empty( $magazines ) ) {
 			return [];
 		}
 		
-		return $magazines;
+		return array_filter($magazines, static function ($magazine) {
+			return is_array( $magazine );
```

Comment:
it might be array even if it is false. Better to check both `is_array( $magazine ) && ! empty( $magazine )`